_posts/2019-03-06-release-1.8.0.md

```diff
@@ -0,0 +1,187 @@
+  Serialization](https://ci.apache.org/projects/flink/flink-docs-master/dev/stream/state/custom_serialization.html)
+  documentation page, as well as the class-level Javadocs of the
+  `TypeSerializer` class.
+
+* **Continuous cleanup of old state based on TTL
+  ([FLINK-7811](https://issues.apache.org/jira/browse/FLINK-7811))**: We
+  introduced TTL (time-to-live) for Keyed state in Flink 1.6
+  ([FLINK-9510](https://issues.apache.org/jira/browse/FLINK-9510)). This
+  feature enabled cleanup and making keyed state entries inaccessible after a
+  defined timeout. In addition state would now also be cleaned up when
+  writing a savepoint/checkpoint.
+
+  Flink 1.8 introduces continous cleanup of old entries for both the RocksDB
+  state backend
+  ([FLINK-10471](https://issues.apache.org/jira/browse/FLINK-10471)) and the heap
+  state backend
+  ([FLINK-10473](https://issues.apache.org/jira/browse/FLINK-10473)). This means
+  that old entries (according to the TTL setting) are continously being cleanup
+  up.
+
+* **SQL pattern detection with user-defined functions and aggregations**: The
+  support of the MATCH_RECOGNIZE clause has been extended by multiple features.
+  User-defined function are now possible which allows for custom logic during
+  pattern detection
+  ([FLINK-10597](https://issues.apache.org/jira/browse/FLINK-10597)).
+  Aggregations allow for more complex CEP definitions
+  ([FLINK-7599](https://issues.apache.org/jira/browse/FLINK-7599)).
+
+  ```
+  SELECT *
+  FROM Ticker
+      MATCH_RECOGNIZE (
+          ORDER BY rowtime
+          MEASURES
+              AVG(A.price) AS avgPrice
+          ONE ROW PER MATCH
+          AFTER MATCH SKIP TO FIRST B
+          PATTERN (A+ B)
+          DEFINE
+              A AS AVG(A.price) < 15
+      ) MR;
+  ```
+
+* **RFC-compliant CSV format ([FLINK-9964](https://issues.apache.org/jira/browse/FLINK-9964))**: The SQL tables can now be read and written in
+  an RFC-4180 standard compliant CSV table format. The format might also be
+  useful for general DataStream API users.
+
+* **New KafkaDeserializationSchema that gives direct access to ConsumerRecord
+  ([FLINK-8354](https://issues.apache.org/jira/browse/FLINK-8354))**: For the
+  Flink `KafkaConsumers`, we introduced a new `KafkaDeserializationSchema` that
+  gives direct access to the Kafka `ConsumerRecord`. This now allows access to
+  all data that Kafka provides for a record, including the headers. This
+  subsumes`KeyedSerializationSchema` functionality, which is deprecated but
+  still available for now.
+
+## Important Changes
+
+* **Changes to bundling of Hadoop libraries with Flink
+  ([FLINK-11266](https://issues.apache.org/jira/browse/FLINK-11266))**:
+  Convenience binaries that include hadoop are no longer released.
+
+  If a deployment relies on `flink-shaded-hadoop2` being included in
+  `flink-dist`, then it must be manually downloaded and copied into the /lib
```

Comment:
"then it must be manually downloaded" -> "then Hadoop must be manually downloaded"